### PR TITLE
Peppy.Osu! version 2021.608.0

### PR DESCRIPTION
**Pull request by aeongdesu**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16770)

### CODE REVIEW

File: manifests/p/Peppy/Osu!/2021.608.0/Peppy.Osu!.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate
+
+PackageIdentifier: Peppy.Osu!
+PackageVersion: 2021.608.0
+PackageLocale: en-US
+Publisher: Dean "peppy" Herbert
```
Comment (OfficialEsco) [inline manifests/p/Peppy/Osu!/2021.608.0/Peppy.Osu!.locale.en-US.yaml:6]:
```suggestion
Publisher: ppy Pty Ltd
Author: Dean "peppy" Herbert
```

File: manifests/p/Peppy/Osu!/2021.608.0/Peppy.Osu!.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate
+
+PackageIdentifier: Peppy.Osu!
+PackageVersion: 2021.608.0
+PackageLocale: en-US
+Publisher: Dean "peppy" Herbert
+PackageName: Osu!
```
Comment (OfficialEsco) [inline manifests/p/Peppy/Osu!/2021.608.0/Peppy.Osu!.locale.en-US.yaml:7]:
```suggestion
PackageName: osu!lazer
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Hey! Since you've been maintaing the Osu! manifest, when did it change to `osu!lazer`?
These changes needs to be added for Osu! to be updated with `winget upgrade`.
image

**Comment by KeepSOBP** [comment]
@OfficialEsco well, it was originally `osu!lazer` `¯\_(ツ)_/¯`

**Review by OfficialEsco — APPROVED** [review]
I swear i've seen it installed as Osu! in windows before, but i haven't played it for 7+ years so who knows 😬
Not your fault, just a quick Quality check with how WinGet detects programs.

**Pull request merged**